src/test/java/org/openstreetmap/atlas/checks/validation/points/InvalidMiniRoundaboutCheckTest.java

```diff
@@ -0,0 +1,164 @@
+package org.openstreetmap.atlas.checks.validation.points;
+
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import org.junit.Assert;
+import org.junit.Rule;
+import org.junit.Test;
+import org.openstreetmap.atlas.checks.configuration.ConfigurationResolver;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.checks.validation.verifier.ConsumerBasedExpectedCheckVerifier;
+
+/**
+ * Tests the {@link InvalidMiniRoundaboutCheck} for each use case.
+ *
+ * @author nachtm
+ */
+public class InvalidMiniRoundaboutCheckTest
+{
+    @Rule
+    public InvalidMiniRoundaboutCheckTestRule setup = new InvalidMiniRoundaboutCheckTestRule();
+
+    @Rule
+    public ConsumerBasedExpectedCheckVerifier verifier = new ConsumerBasedExpectedCheckVerifier();
+
+    @Test
+    public void configurableDoesNotFlagLowValence()
```

Comment:
All unit test function names should start with `test`.  So, `testConfigurable....`